src/VirtualList/VirtualList.js

```diff
@@ -210,6 +215,16 @@ VirtualList.getItems = function (viewTop, viewHeight, listTop, itemHeight,
   return result;
 };
 
+VirtualList.getItemsToRender = function (props, state) {
```

Comment:
Why is this a _static_ method? Let's turn this one into a public method or an util/private helper.